x/pot/handler.go

```diff
@@ -122,6 +122,15 @@ func handleMsgSlashingResourceNode(ctx sdk.Context, k keeper.Keeper, msg types.M
 		}
 	}
 
-	err := k.SlashingResourceNode(ctx, msg.NetworkAddress, msg.WalletAddress, msg.Slashing, msg.Suspend)
-	return &sdk.Result{}, err
+	amt, nodeType, err := k.SlashingResourceNode(ctx, msg.NetworkAddress, msg.WalletAddress, msg.Slashing, msg.Suspend)
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeSlashing,
+			sdk.NewAttribute(types.AttributeKeyWalletAddress, msg.WalletAddress.String()),
+			sdk.NewAttribute(types.AttributeKeyNodeP2PAddress, msg.NetworkAddress.String()),
+			sdk.NewAttribute(types.AttributeKeyAmount, amt.String()),
+			sdk.NewAttribute(types.AttributeKeySlashingNodeType, nodeType.String()),
```

Comment:
Still needs an attribute for the suspend boolean. I wouldn't want to mark a PP node as suspended in SP if it's not suspended in stchain

Reply:
updated